docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -806,4 +806,163 @@ The following is performed on ArduCopter version v4.5.7
 
 .. image:: ../_static/manual_images/dronecan/ardupilot_dna_success.PNG
 
-6. Now, if you reboot your module, and check the DroneCAN configuration window again, you'll find that ArduPilot allocates the same ID again
\ No newline at end of file
+6. Now, if you reboot your module, and check the DroneCAN configuration window again, you'll find that ArduPilot allocates the same ID again
+
+*************************************
+Calculating DroneCAN Utilization
+*************************************
+
+Background
+============
+The basic premise of calculating DroneCAN bus utilization is to figure out how much time you spend sending DroneCAN messages compared to some period of time. For example, 
+suppose your message takes 500ms to transmit. Then, in a 1 second period of time, that message takes up 50% of the available bandwidth.
+
+The basic calculation we are going to perform is summarized in the image below:
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/dronecan_utilization_calculation.png
+
+DroneCAN frames are broken down as follows:
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/dronecan_packet.png
+
+Please note that the Data bytes may be split further depending on whether or not the current frame is the beginning of a multi frame transfer. As specified by the 
+`DroneCAN specification <https://dronecan.github.io/Specification/4.1_CAN_bus_transport_layer/#:~:text=is%20not%20suppressed.-,Payload,-The%20Data%20field>`_, the data may be broken down as either: 
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/single_frame_transfer.png
+
+or
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/multi_frame_transfer.png
+
+With no data (and thus no tail byte), this is 67 bits and 134us, and with a full 7 bytes of data (plus tail byte), this is 131 bits or 256us.
+
+You can read more about the DroneCAN transport layer in the `DroneCAN spec <https://dronecan.github.io/Specification/4.1_CAN_bus_transport_layer/>`_.
```

Comment:
Might be better to say "specification" instead of "spec", just for avoiding casual shortenings